kubernetes/services/middleman.yml.j2

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
+kind:       Deployment
+metadata:
+  name: middleman
+spec:
+  replicas:             1
+  revisionHistoryLimit: 1
+  template:
+    metadata:
+      labels:
+        run: middleman
+    spec:
+      containers:
+        - name:             concent-api
+          image:            {{ docker_registry }}/{{image_prefix }}concent-api:{{ image_version }}
+          imagePullPolicy:  Always
+          command:          ["/usr/local/bin/concent-api-manage.sh", "middleman", "--bind-address", "0.0.0.0", "--internal-port", "9054"]#FIXME AFTER external-port was added, "--external-port 9055"]
```

Comment:
This FIXME needs to be resolved before we can merge it.

Reply:
Done.